### ISSUE TITLE
Language modelling head weights

### ISSUE THREAD

**Issue by tdsone** [MAGICS-LAB/DNABERT_S#26]
Hey there, 

great effort on the repo! I was curious where I would find the weights of the trained language modelling head? I.e. the one you used to predict the masks that converts your embeddings into logits?

Cheers
Timon

**Comment by Zhihan1996** [comment]
Sorry for this super late reply! Missed this question earlier.

DNABERT-S is trained with contrastive learning instead of language modeling, so we didn't save it's language modeling head. But if you are interested, you could achieve the language modeling head of DNABERT-2 but loading it with:

```
from transformers import AutoModelForMaskedLM

model = AutoModelForMaskedLM.from_pretrained("zhihan1996/DNABERT-2-117M", trust_remote_code=True)
```
